Assets/Scripts/Lever.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+public class Lever : MonoBehaviour {
+    public UnityEvent leftOption;
+    public UnityEvent rightOption;
+    private bool left;
+    private Animator animator;
+    private bool inRange;
+
+    void Awake()
+    {
+        left = true;
+        inRange = false;
+        animator = gameObject.GetComponent<Animator>();
+    }
+
+    private void Start()
+    {
+        leftOption.Invoke();
+    }
+
+    private void FixedUpdate()
+    {
+        // TODO: Change to interact button
+        if (inRange == true && Input.GetKeyDown("q"))
```

Comment:
How are you going to know that the object inside the lever's collider is indeed a player? How are you going to distinguish between the two players?